src/utils/formHelperFunctions.js

```diff
@@ -464,6 +464,28 @@ function shouldElementHide({ fields, values, element }) {
         : !matchValues.includes(value);
 }
 
+/**
+ * Files coming from Feathery servers are provided as objects { path, url }
+ * We need to convert the url into a blob for standardized handling
+ * We recurse to handle repeated fields
+ */
+async function convertToBlob(maybeBlob) {
```

Comment:
if we instead do conversion of only the session data fetched from feathery servers, we know all of those are urls and no need to do hacky type detection here